default/stringtables/en.txt

```diff
@@ -4277,14 +4277,18 @@ Robotic species are sapient machines. These species' max population can be incre
 PHOTOTROPHIC_SPECIES_TITLE
 Phototrophic Metabolism
 PHOTOTROPHIC_SPECIES_TEXT
-'''Phototrophic species live mainly or entirely on sunlight. These species' max population is greatly influenced by the brightness of the local star - the bigger the better. Growth specials do not benefit them.
+'''Phototrophic species live mainly or entirely on sunlight. These species' max population is greatly influenced by the brightness of the local star and the planet size - the brighter or bigger, the better. Growth specials do not benefit them.
 
 Population changes:
-Blue Star: Tiny +3 , Small +6 , Medium +9 , Large +12 , Huge +15.
-White Star: Tiny +1.5 , Small +3 , Medium +4.5 , Large +6 , Huge +7.5.
-Yellow Star, Orange Star: No changes.
-Red Star, Neutron Star: Tiny -1 , Small -2 , Medium -3 , Large -4 , Huge -5.
-Black Hole, No Star: Tiny -10 , Small -20 , Medium -30 , Large -40 , Huge -50.'''
+* [[STAR_BLUE]] star: ([[SZ_TINY]] +3) ([[SZ_SMALL]] +6) ([[SZ_MEDIUM]] +9) ([[SZ_LARGE]] +12) ([[SZ_HUGE]] +15)
```

Comment:
On the other hand, it would make a pretty long vertical list as there are 5 categories, and 5 items per categories + the blank lines = approximatively 30 lines.

The purpose is to make possible to compare easily the values in one glance (or two :p)

But no worries, if the Roboto PR is approved and merged, the list looks good like this (not so much with DejaVuSans).

As I have to fix this PR conflict, I'm even going to remove the extra blank lines in the list (only in Phototropic Metabolism article)